install/cameras.csv

```diff
@@ -37,6 +37,7 @@ Mobotix AG,M15,10.18.168.61,WICF,01,0,0,0,0,0,2016-06-01T23:40:00Z,9999-01-01T00
 Mobotix AG,M15,10.20.210.120,KMTP,01,9.5,290,-2.5,0,0,2020-05-07T00:20:00Z,2022-04-12T07:40:01Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
 Mobotix AG,M15,10.20.245.115,TOKR,01,0,0,0,0,0,2018-05-03T00:00:00Z,9999-01-01T00:00:00Z,Karewerewa looking at TeMaari
 Mobotix AG,M15,10.7.46.153,TOKR,01,0,0,0,0,0,2017-10-12T00:40:00Z,2018-05-02T23:55:00Z,
+Mobotix AG,M16,10.26.194.146,MTSR,01,12.5,37,2,0,0,2022-09-23T02:40:00Z,9999-01-01T00:00:00Z,Mangateitei looking at South Ruapehu
```

Comment:
The height is said to be -2 in the ticket comment. but is +2 here. I figure this is correct, as generally up is positive with height. Is the ticket comment meaning depth is -2?

Previous install doesn't have 12.5 dip, despite ticket comment saying it did.

Reply:
Don't know why the ticket says -2 to be honest but I know the camera has to be above the ground surface!

Reply:
@rumachan @staylorofford clarifying why I put -2, generally seismometers were given positive values to indicate depth, hence why I chose a negative depth (above ground).